_posts/2020-04-06-swagger-is4.md

```diff
@@ -1,9 +1,280 @@
 ---
 layout: post
-title:  "Integrate swagger with Identity Server 4"
-date: 2020-04-06 08:00:00 +0100
+title:  "Integrate Swagger with Identity Server 4 (.NET Core 3.1)"
+date: 2020-04-06 09:30:00 +0100
 categories: [development]
 tags: [swagger, net core, Identity Server 4]
 ---
 
-Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin nibh augue, suscipit a, scelerisque sed, lacinia in, mi. Cras vel lorem. Etiam pellentesque aliquet tellus. Phasellus pharetra nulla ac diam. Quisque semper justo at risus. Donec venenatis, turpis vel hendrerit interdum, dui ligula ultricies purus, sed posuere libero dui id orci. Nam congue, pede vitae dapibus aliquet, elit magna vulputate arcu, vel tempus metus leo non est. Etiam sit amet lectus quis est congue mollis. Phasellus congue lacus eget neque. Phasellus ornare, ante vitae consectetuer consequat, purus sapien ultricies dolor, et mollis pede metus eget nisi. Praesent sodales velit quis augue. Cras suscipit, urna at aliquam rhoncus, urna quam viverra nisi, in interdum massa nibh nec erat.
\ No newline at end of file
+## While building API in microservices architecture, the best way to create clear documentation with a friendly GUI is Swagger, which also allows execution of API tests. Running Swagger with API is quite easy and well-documented, but when API is protected by OAuth, it can be tricky to integrate them. In this article, I'd like to show you how to run a sample .NET Core 3.1 API with Swagger (Swashbuckle for .NET) and Swagger Authroziation via Identity Server 4.
```

Comment:
Maybe add a hyperlink to the Identity Server 4 text that points to https://github.com/IdentityServer/IdentityServer4?

Reply:
ok, fixed :)